payment-cards/avs.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: AVS
+permalink: /payment-cards/avs/
+---
+
+#Address Verification System (AVS)
+
+The Address Verification System (AVS) is a system used to verify the address of a person claiming to own a credit card. The system will check the billing address of the credit card provided by the user with the address on file at the credit card company. The other security features for the credit card include the CVV2 number.
```

Comment:
Would love it if CVV2 linked to a glossary entry. I can do that once this is merged.